src/Anyt/BugTrackerBundle/DoctrineExtensions/DBAL/Types/UTCDateTimeType.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+namespace Anyt\BugTrackerBundle\DoctrineExtensions\DBAL\Types;
+
+use Doctrine\DBAL\Types\DateTimeType;
+use Doctrine\DBAL\Platforms\AbstractPlatform;
+use Doctrine\DBAL\Types\ConversionException;
+
+class UTCDateTimeType extends DateTimeType
```

Comment:
Not needed https://github.com/laboro/platform/blob/master/src/Oro/Bundle/LocaleBundle/DoctrineExtensions/DBAL/Types/UTCDateTimeType.php#L9-9